oktapam/resource_ad_user_sync_task_settings.go

```diff
@@ -227,3 +228,21 @@ func importADUserSyncTaskSettingsState(ctx context.Context, d *schema.ResourceDa
 	d.SetId(adUserSyncTaskSettingsID)
 	return []*schema.ResourceData{d}, nil
 }
+
+func resourceADUserSyncTaskSettingsCustomizeDiff(ctx context.Context, diff *schema.ResourceDiff, v interface{}) error {
+	if diff.HasChanges(attributes.Frequency, attributes.StartHourUTC, attributes.IsActive) {
+		Frequency := diff.Get(attributes.Frequency).(int)
+		StartHourUTC := diff.Get(attributes.StartHourUTC).(int)
+		IsActive := diff.Get(attributes.IsActive).(bool)
+		if IsActive && Frequency < 1 {
```

Comment:
Do we need to say production here or just say "frequency must be specified when scheduling AD User Sync Task"

Reply:
Yes, I will replace production with active